mrp_production_estimated_cost/README.rst

```diff
@@ -0,0 +1,96 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Estimated costs in manufacturing orders
+=======================================
+
+Thanks to this module, you can estimate the cost of your manufacturing orders
```

Comment:
This module extends the functionality of MRP adding estimates for Manufacturing Orders costs, as followed: